manifests/REFERENCE.md

```diff
@@ -0,0 +1,6 @@
+# Reference
```

Comment:
I think this file was likely accidentally created and should be deleted.

Reply:
Deleted.